docs/tutorials/go-built-in.md

```diff
@@ -98,6 +98,10 @@ func NewKVStoreApplication() *KVStoreApplication {
 	return &KVStoreApplication{}
 }
 
+func (KVStoreApplication) SetOption(abcitypes.RequestSetOption) abcitypes.ResponseSetOption {
```

Comment:
I think guides on master should reflect master and versioned docs should reflect the version. This tutorial is different from java and out of process one's now.

Reply:
ok